### ISSUE TITLE
Some improvement before interview

### ISSUE THREAD

**Issue by rfelcman** [tvallin/StreamCombiner#1]
Hello I am member of Dmitry team and I have following comments for Your project:
1. EclipseLink used, but little bit old 2.5.0-RC1 (why RC1?). Check the latest. (groupId, artifactId is correct).
2. JUnit version same as point 1. Major version 4 or 5 is up to You.
3. No JUnit test (empty class there).
4. Can't build project with JDK 8 and higher. Please use JDK 8 or JDK 11.
5. [Low priority] Broken formating in sources
6. Missing usage description or some Javadoc. Some nice JUnit can describe usage too.

**Comment by thibaultvallin** [comment]
Hello, thank you very much for your helpfull comment.

1 and 2. The modification had been made
3. I am going to update the Junit test
4. I need to investigate because from my side, I build the project with JDK 11 or 12 and it is working
6. I am going to update files with complete Javadoc + Description.

Next update the August 15th in the afternoon

**Comment by rfelcman** [comment]
About point #4. Which version of Maven are You using? And JDK. Oracle JDK or Open JDK?

**Comment by thibaultvallin** [comment]
I am using JDK from oracle downloaded on the company website.
The Maven version is 3.6.1.

**Comment by senivam** [comment]
regarding #4 for me it started to work only after adding to pom:
```
<build>
        <plugins>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.8.1</version>
                <configuration>
                    <source>1.8</source>
                    <target>1.8</target>
                </configuration>
            </plugin>
        </plugins>
    </build>
```
valid both for JDK 8 and 11.

my maven is 3.5.3

**Comment by senivam** [comment]
so, did you fix that?

**Comment by tvallin** [comment]
Yes I did :)
